src/libs/SidebarUtils.ts

```diff
@@ -98,7 +98,9 @@ function getOrderedReportIDs(
         const isFocused = report.reportID === currentReportId;
         const allReportErrors = OptionsListUtils.getAllReportErrors(report, reportActions) ?? {};
         const hasErrorsOtherThanFailedReceipt = doesReportHaveViolations || Object.values(allReportErrors).some((error) => error?.[0] !== 'report.genericSmartscanFailureMessage');
-        const shouldOverrideHidden = hasErrorsOtherThanFailedReceipt || isFocused || report.isPinned;
+
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        const shouldOverrideHidden = hasErrorsOtherThanFailedReceipt || isFocused || report.isPinned || ReportUtils.isSystemChat(report);
```

Comment:
Do we still need to disable eslint if we save `ReportUtils.isSystemChat(report)` in a variable?